messagingpush/src/main/java/io/customer/messagingpush/CustomerIOFirebaseMessagingService.kt

```diff
@@ -86,8 +86,19 @@ open class CustomerIOFirebaseMessagingService : FirebaseMessagingService() {
             handleNewToken(context = context, token = token)
         }
 
+        /**
+         * Gets initialized instance of SDK. If the SDK is not initialized, it
+         * attempts to initialize with messaging push module so the module classes
+         * are initialized as well.
+         */
+        private fun getSDKInstanceOrNull(context: Context): CustomerIO? = try {
+            CustomerIO.instanceOrNull(context, listOf(ModuleMessagingPushFCM()))
+        } catch (e: Exception) {
+            null
+        }
```

Comment:
We can probably remove the try-catch block, as the `instanceOrNull` method inside it already handles the exceptions? 

```suggestion
fun getSDKInstanceOrNull(context: Context): CustomerIO? {
    return CustomerIO.instanceOrNull(context, listOf(ModuleMessagingPushFCM()))
}
```

Reply:
Oops 🤦🏻 Yeah 😅